### PR TITLE
adding in command line parameters from vtctl.go

### PR DESCRIPTION
**Pull request by hallaroo**
Doing the manual migration requested [here](https://github.com/vitessio/website/issues/498)

### CODE REVIEW

File: content/en/docs/reference/programs/vtctl/keyspaces.md [hunk 1]
```diff
@@ -171,7 +143,8 @@ Changes the ServedFromMap manually. This command is intended for emergency fixes
 
 #### Example
 
-<pre class="command-example">SetKeyspaceServedFrom [-source=&lt;source keyspace name&gt;] [-remove] [-cells=c1,c2,...] &lt;keyspace name&gt; &lt;tablet type&gt;</pre>
+<pre class="command-example">SetKeyspaceServedFrom [-source=&lt;source keyspace name&gt;] [-remove] [-cells=c1,c2,...] &lt;keyspace name&gt; &lt;tablet type&gt;
+Changes the ServedFromMap manually. This command is intended for emergency fixes. This field is automatically set when you call the *MigrateServedFrom* command. This command does not rebuild the serving graph.</pre>
```
Comment (deepthi) [inline content/en/docs/reference/programs/vtctl/keyspaces.md:147]: we are no longer documenting `MigrateServedFrom` but this command refers to it.
@sougou @rohit-nayak-ps we should update the help-text in code and provide it to @hallaroo to edit the website help text.

**Review by deepthi — APPROVED** [review]
LGTM.
You can address the one comment before merging, or do that in a later PR.

File: content/en/docs/reference/programs/vtctl/keyspaces.md (hunk 1 above)
Comment (hallaroo) [reply content/en/docs/reference/programs/vtctl/keyspaces.md:147]: @deepthi - I'd love to address the comment in this review. For the `MigrateServedFrom` I noticed that is was in this document twice (e.g. duplicated) so I removed just one reference. 

There should be information on `MigratedServedFrom` starting on line 325. I kept the one that was next to `MigrateServedTypes`. Does that fix the comment or do we need to update the text there?

Perhaps I should also add a link into the text on 147?

Comment (deepthi) [reply content/en/docs/reference/programs/vtctl/keyspaces.md:147]: This sounds fine. A link is nice-to-have. Do you want to add that before we merge?

Comment (hallaroo) [reply content/en/docs/reference/programs/vtctl/keyspaces.md:147]: Yes I'll add the link before we merge so that it's all set.

**Pull request merged**